src/gurt/dlog.c

```diff
@@ -296,6 +310,55 @@ static void dlog_cleanout(void)
 #endif
 }
 
+void *d_log_tls_get()
+{
+	if (!mst.tls_key_created)
+		return NULL;
+
+	return pthread_getspecific(mst.tls_key);
+}
+
+int d_log_tls_init()
```

Comment:
(style)  Bad function definition - int d_log_tls_init() should probably be int d_log_tls_init(void)